### PR TITLE
fix: layout button in main page.

### PR DESCRIPTION
**Pull request by GlistenSTAR**
- add css for layout button in main page

### CODE REVIEW

File: src/www/+page.svelte [hunk 1]
```diff
@@ -55,7 +55,7 @@
 				>Components</Button
 			>
 			&nbsp;
-			<Button href="{base}/docs/layouts/card" size="xl">Layouts</Button>
+			<Button href="{base}/docs/layouts/card" size="xl" id="layout_button">Layouts</Button>
```
Comment (blokhin) [inline src/www/+page.svelte:58]: @GlistenSTAR are you sure Svelte allows you to setup the `id`? I thought, it's recommended to use classes, but please correct me if I'm wrong.

**Review by blokhin — CHANGES_REQUESTED** [review]
please, see my comments

File: src/www/docs/components/periodictable/+page.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 file: periodictable.md
 title: Periodic Table
 api: [
-	{title: 'selected: string[] = []', description: 'Selected element array output', variables: '[]'},
+	{title: 'selected: string[] = []', description: 'Selected element array output', variables: 'any string'},
```
[the remark this answers is not in the document]
Comment (blokhin) [reply src/www/docs/components/periodictable/+page.md:5]: It even makes sense to mention them all as the `ENUM` type

[the remark this answers is not in the document]
Comment (GlistenSTAR) [reply src/www/docs/components/periodictable/+page.md:5]: `ENUM`
I agreed.

File: src/www/+page.svelte (hunk 1 above)
Comment (GlistenSTAR) [reply src/www/+page.svelte:58]: yes, id is possible to change the css.
We can use it.

File: src/www/docs/components/periodictable/+page.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (GlistenSTAR) [reply src/www/docs/components/periodictable/+page.md:5]: For `Enum`
['period 1', 'period 2', 'period 3', 'period 4', 'period 5', 'period 6', 'period 7', 'lanthanide', 'actinide', 'alkali', 'alkaline', 'group 3', 'group 4', 'group 5', 'group 6', 'group 7', 'group 8', 'group 9', 'group 10', 'group 11', 'group 12', 'triels', 'tetrels', 'pnictogen', 'chalcogen', 'halogen', 'noble gas'],
We will use this. How about?

[the remark this answers is not in the document]
Comment (GlistenSTAR) [reply src/www/docs/components/periodictable/+page.md:5]: if `Enum` is complex, we will remain with `[]`.
How about?

**Review by GlistenSTAR — COMMENTED** [review]
I have done review code

**Review by blokhin — APPROVED** [review]

**Comment by blokhin** [comment]
@GlistenSTAR :ok_hand:

**Pull request merged**